sherpa/astro/tests/test_data.py

```diff
@@ -859,3 +859,23 @@ def test_arf_rmf_get_x(make_data_path):
 
     np.testing.assert_array_almost_equal(expected_array_10, actual_arf_10)
     np.testing.assert_array_almost_equal(expected_array_10, actual_rmf_10)
+
+
+def test_arf_rmf_get_x_unit():
+
```

Comment:
I think this would be better as two tests - one for arf and one for rmf. It's a small thing, but means we can ensure we test both code paths in case the ARF one fails. Given that the code paths are essentially identical I could imagine parameterising it with the arrays/function to call (or other means of reducing code duplication).

Reply:
Fair enough. However I think the (small) code duplication is a reasonable price to pay to have the tests more readable, so I won't parameterize them